src/evict/evict_lru.c

```diff
@@ -1313,6 +1313,8 @@ __evict_lru_walk(WT_SESSION_IMPL *session)
 		__evict_list_clear(session, &queue->evict_queue[--entries]);
 
 	queue->evict_entries = entries;
+	WT_STAT_CONN_INCRV(
+	    session, cache_eviction_pages_queued_post_lru, entries);
```

Comment:
@sulabhM 
I see that we have `queue->evict_candidates` to track the number of things to be evicted whereas I'm tracking `queue->evict_entries` which is the overall size of the queue. Do you want me to also add a candidates statistic? Or is it not that interesting.

Reply:
I think `entries` here is not interesting. I meant to track `queue->evict_candidates` after we make adjustments with the if/else block that starts at 1332. Those adjustments are that bite us at times with FTDC when we do not know how many pages we actually decided to evict.

Reply:
2b0df56